requirements.txt

```diff
@@ -1,13 +1,13 @@
 # This pulls in all the libraries needed to run Selenium tests
-# on Mozilla WebQA projects
+# on Mozilla Web QA projects
```

Comment:
Remove the comment, we've been doing this in other repos as part of the cleanup.